doc/classes/NavigationMeshSourceGeometryData2D.xml

```diff
@@ -29,12 +37,30 @@
 				Clears the internal data.
 			</description>
 		</method>
+		<method name="clear_projected_obstructions">
+			<return type="void" />
+			<description>
+				Clears all projected obstructions.
+			</description>
+		</method>
 		<method name="get_obstruction_outlines" qualifiers="const">
 			<return type="PackedVector2Array[]" />
 			<description>
 				Returns all the obstructed area outlines arrays.
 			</description>
 		</method>
+		<method name="get_projected_obstructions" qualifiers="const">
+			<return type="Array" />
+			<description>
+				Returns the projected obstructions as an Array of Dictionaries with the following key value pairs:
+				[codeblocks]
+				[gdscript]
+				"vertices" : PackedFloat32Array
+				"carve" : bool
+				[/gdscript]
+				[/codeblocks]
```

Comment:
I would not use codeblocks for this purpose.
This applies to the other arrays of dictionaries in this PR as well

Admittedly, the way these lists of entries are formatted is a bit inconsistent across the whole class reference. But what those values are could be explained a bit, roughly too
```suggestion
				Returns the projected obstructions as an [Array] of dictionaries with the following key value pairs:
				- [code]vertices[/code] - A [PackedFloat32Array] [...];
				- [code]carve[/code] - A [bool] [...].
```

Reply:
Reasoning for code blocks is because the names or even expected types are different in other languages e.g. C# and that trips users into using wrong types all the time. I have C# users reporting navigation "bugs" all the time because PackedXYZArrays and other stuff require different type use in C# or else things turn null. Now I am dont use X-language , I dont know what will be correct for the documentation, so I omitted that part to be added by others that search the documentation for codeblocks that have only gdscript added.